assets/stylesheets/kitten/utilities/text/_weight.scss

```
@@ -8,23 +8,26 @@
 ///
 /// @example html
 ///
-///     <p class="k-u-weight-light">…</p>
-///     <p class="k-u-weight-regular">…</p>
-///     <p class="k-u-weight-bold">…</p>
+///     <p class="k-u-weight-400">…</p>
+///     <p class="k-u-weight-500">…</p>
+///     <p class="k-u-weight-700">…</p>
 
 @mixin k-utilities-weight {
-  .k-u-weight-light {
+  .k-u-weight-light,
+  .k-u-weight-400 {
     font-family: map-get(k-map-fetch($k-fonts, 'light'), 'family') !important;
     font-weight: map-get(k-map-fetch($k-fonts, 'light'), 'weight') !important;
```

Regarde si ça ne marcherait pas avec 
```suggestion
    font-family: map-get(k-map-fetch($k-fonts, '400'), 'family') !important;
    font-weight: map-get(k-map-fetch($k-fonts, '400'), 'weight') !important;
```

---

Non ça ne fonctionne pas ! j'ai une erreur